src/app/components/Redirect404/Redirect404.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import DocumentTitle from 'react-document-title';
+import PropTypes from 'prop-types';
+import appConfig from '../../data/appConfig';
+
+const Redirect404 = (props, context) => {
+
+  const originalUrl = context &&
+    context.router &&
+    context.router.location &&
+    context.router.location.query &&
+    context.router.location.query.originalUrl;
+
+  const {
+    circulatingCatalog,
+    classicCatalog,
+  } = appConfig;
+
+  return (
+    <div className="redirect404">
```

Comment:
Definitely add the `.nypl-ds` class to get the right fonts.

Reply:
Changed